src/Api/Controllers/MergedIndexController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace FoF\Linguist\Api\Controllers;
+
+use Flarum\Api\Controller\AbstractListController;
+use Flarum\Http\RequestUtil;
+use FoF\Linguist\Api\Serializers\StringKeySerializer;
+use FoF\Linguist\Repositories\DefaultStringsRepository;
+use FoF\Linguist\Repositories\StringRepository;
+use FoF\Linguist\TextString;
+use Illuminate\Support\Arr;
+use Psr\Http\Message\ServerRequestInterface;
+use Tobscure\JsonApi\Document;
+
+class MergedIndexController extends AbstractListController
+{
+    public $serializer = StringKeySerializer::class;
+
+    /**
+     * @var DefaultStringsRepository
+     */
+    protected $defaultStrings;
+
+    /**
+     * @var StringRepository
+     */
+    protected $strings;
+
+    public function __construct(
+        DefaultStringsRepository $defaultStrings,
+        StringRepository $strings
+    ) {
+        $this->defaultStrings = $defaultStrings;
+        $this->strings = $strings;
+    }
+
+    protected function data(ServerRequestInterface $request, Document $document)
+    {
+        RequestUtil::getActor($request)->assertCan('viewStringKeys');
+
+        // Extract filters from the request.
+        $filters = $this->extractFilter($request);
+        $filter = $this->defaultStrings->getPrefix($filters);
+
+        // Retrieve all translations from the default repository and returns a Collection.
+        // @example
+        //  array:2 [▼
+        // "key" => "glowingblue-integrable.ui.discussions.show_more"
```

Comment:
Can we maybe get a more generic example here?

```suggestion
        // "key" => "acme-foobar.forum.example_string"
```